src/python/examples/memory_growth_test.py

```diff
@@ -62,14 +60,19 @@
     input0_data = np.arange(start=0, stop=16, dtype=np.int32)
     input0_data = np.expand_dims(input0_data, axis=0)
 
-    for i in range(FLAGS.repetitions):
-        if FLAGS.protocol.lower() != "grpc":
-            triton_client = httpclient.InferenceServerClient(
-                url="localhost:8000", verbose=FLAGS.verbose)
-        else:
-            triton_client = grpcclient.InferenceServerClient(
-                url="localhost:8001", verbose=FLAGS.verbose)
+    if FLAGS.protocol.lower() == "grpc":
```

Comment:
I think Guan was recommending to move the input/output setup code here too, but it can be added afterwards if we just want to get the version upgrade in since this is just in a test.